build/ci-build.yml

```diff
@@ -89,6 +91,18 @@ stages:
             inputs:
               artifact: 'Build'
               path: '$(Build.SourcesDirectory)'
+          - task: Docker@1
+            displayName: 'Build Docker image from newly created $(ProjectName) project from template'
+            inputs:
+              dockerFile: ./src/$(ProjectName)/Dockerfile
+              imageName: '$(ProjectName):$(Build.BuildId)'
+          - task: Docker@1
+            displayName: 'Run new  project Docker image'
+            inputs:
+              command: 'Run an image'
+              imageName: '$(ProjectName):$(Build.BuildId)'
+              containerName: '$(ProjectName)'
+              ports: '5000'
           - template: test/run-integration-tests.yml@templates
```

Comment:
I've found it usefule to run this step after running the tests: https://github.com/tomkerkhove/promitor/blob/master/build/azure-devops/agents-ci-discovery.yml#L141

Note that it always runs,even if the tests fail. That way you can use the logs during troubleshooting.

Reply:
Added! Thanks.